DatabaseServer/database_server.py

```diff
@@ -310,6 +310,15 @@ def _get_iocs_not_to_stop() -> list:
         """
         return IOCS_NOT_TO_STOP
 
+@retry(max_attempts=5, interval=10, exception=Exception)
```

Comment:
Does `max_attempts` of 5 actually make sense here? This process can't do anything useful until it establishes a connection, so it feels more like we should be waiting indefinitely for the connection to become available.

Or if you think the limit of 5 is justified, then we need to handle the case where it never managed to connect properly (see my other comment a few lines below)